src/util/tubemap.js

```diff
@@ -723,12 +724,15 @@ function placeReads() {
           readIndex: idx,
           pathIndex: pathIdx,
           previousY: previousValidY,
+          previousNode: previousValidNode
         });
       }
     });
   });
 
-  elementsWithoutNode.sort(compareNoNodeReadsByPreviousY);
+  console.log("elementsWithoutNode before sort:", elementsWithoutNode)
+  elementsWithoutNode.sort(compareNoNodeReads);
+  console.log("elementsWithoutNode:", elementsWithoutNode)
```

Comment:
I don't think we need this logging in the final version.
```suggestion
  elementsWithoutNode.sort(compareNoNodeReads);
```